tests/endtoend/blob_functions/put_get_multiple_blobs_as_bytes_return_http_response/main.py

```diff
@@ -15,22 +15,22 @@ def _generate_content_and_digest(content_size):
 
 def main(
         req: azf.HttpRequest,
-        input_file_1: bytes,
-        input_file_2: bytes,
-        output_file_1: azf.Out[bytes],
-        output_file_2: azf.Out[bytes]) -> azf.HttpResponse:
+        inputfile1: bytes,
```

Comment:
I started hitting issues with the use of `_` in the binding name. Removed the `_` for resolving the tests. Doesn't affect the test.